openrest4j-api/src/main/java/com/openrest/v1_1/ContactlessDineIn.java

```diff
@@ -0,0 +1,42 @@
+package com.openrest.v1_1;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+
+import java.io.Serializable;
+import java.util.*;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class ContactlessDineIn implements Serializable, Cloneable {
+    private static final long serialVersionUID = 1L;
+
+    /** Default constructor for JSON deserialization. */
+    public ContactlessDineIn() {}
+
+    public ContactlessDineIn(String comment) {
+        this.comment = comment;
+    }
+
+    @Override
+    public ContactlessDineIn clone() {
+        return new ContactlessDineInInfo(this.comment);
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        ContactlessDineIn that = (ContactlessDineIn) o;
+        return Objects.equals(comment, that.comment);
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(this.comment);
+    }
+
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    public String comment;
```

Comment:
not sure about the comment naming, so if u have some idea please share - this field will represent the label input of the user